nailgun/entities.py

```diff
@@ -2472,6 +2479,24 @@ def create_payload(self):
         """
         return {u'filter': super(Filter, self).create_payload()}
 
+    def read(self, entity=None, attrs=None, ignore=None, params=None):
+        """Deal with different named data returned from the server
+        """
+        if ignore is None:
+            ignore = set()
+        if attrs is None:
+            attrs = self.read_json()
+            attrs['override'] = attrs.pop('override?')
```

Comment:
This 2 lines and 2 in the `else` scope looks identical. Maybe you can just move them outside of the `if` scope and remove `else` part?